src/qibolab/instruments/qm/ports.py

```diff
@@ -140,12 +140,39 @@ class OctaveOutput(QMOutput):
 
     Can be external or internal.
     """
-    output_mode: str = field(default="always_on", metadata={"config": "output_mode"})
+    output_mode: str = field(default="triggered", metadata={"config": "output_mode"})
     """Can be: "always_on" / "always_off"/ "triggered" / "triggered_reversed"."""
+    digital_delay: int = 57
```

Comment:
You could define 57 as a top variable constant

Reply:
https://github.com/qiboteam/qibolab/pull/856/commits/0d6c4b5555279e57abcf5e22914a78dfe1b82257